java-ast/addon/src/main/java/org/jboss/windup/ast/java/ReferenceResolvingVisitor.java

```diff
@@ -1361,7 +1463,20 @@ public PackageAndClassName(String packageName, String className)
             this.packageName = packageName;
             this.className = className;
         }
+        
+        public String toString()
+        {
+            StringBuffer sb = new StringBuffer();
+            if (this.packageName != null)
+            {
+                sb.append(this.packageName).append(".");
+            }
+            if (this.className != null)
+                sb.append(this.className);
 
+            return sb.toString();
+        }
+        
         public static PackageAndClassName parseFromQualifiedName(String qualifiedName)
         {
             final String packageName;
```

Comment:
(Not related to the PR) This class seriously needs to be refactored to smaller parts.